<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code Typing Trainer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1 Minute</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Minute</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Minute</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="timerIcon">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>87</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SelectTime">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>511</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select limit time</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start!</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
